docs/typescript/how-to-set-a-heartbeat-timeout-in-typescript.md

```diff
@@ -0,0 +1,21 @@
+---
+id: how-to-set-a-heartbeat-timeout-in-typescript
+title: How to set a Heartbeat Timeout in Typescript
+sidebar_label: Set a Heartbeat Timeout
+description: Set a Heartbeat Timeout
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+To set a Heartbeat Timeout, create an instance of `longRunningActivity` and set the `heartbeatTimeout` to a time that the Activity can take to complete. If it takes longer than that, the Activity fails.
```

Comment:
```suggestion
To set a Heartbeat Timeout, use [`ActivityOptions.heartbeatTimeout`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions#heartbeattimeout). If the Activity takes longer than that between heartbeats, the Activity is failed.
```
Is there a language-agnostic definition of Heartbeat Timeout that shows up in the Guide before this snippet?